.github/workflows/publish-release.yml

```diff
@@ -7,10 +7,22 @@ on:
         required: true
 
 jobs:
+  is-release:
```

Comment:
This check is handled by `main.yml`. By the time we get here, we know that we want to trigger a release. This is another part of the restructuring that went into `main.yml`.

Reply:
removed [ee54889](https://github.com/MetaMask/eth-simple-keyring/pull/125/commits/ee54889bdf8168120d9ccb57bd970d35d6e2394d). Is it accessing the `IS_RELEASE` value correctly now?

Reply:
Nope. That is set based on the job `is-release`, which no longer exists. Checkout the "Checks" tab here on this PR, and you should see a workflow error explaining this issue.

Reply:
fixed [here](https://github.com/MetaMask/eth-simple-keyring/pull/125/commits/955da279760e184b02ba4d71155a122b99b26009) (I believe)